sdk/identity/azure-identity/README.md

```diff
@@ -0,0 +1,73 @@
+# Azure SDK Identity Library for C++
+
+Azure::Identity (`azure-identity`) provides shared primitives, abstractions, and helpers for modern Azure SDK client libraries written in the C++. These libraries follow the [Azure SDK Design Guidelines for C++][azure_sdk_cpp_development_guidelines].
+
+The library contains commonly (but not universally) used credential types.
+
+## Getting started
+
+Typically, you will not need to download `azure-identity`; it will be downloaded for you as a dependency of the client libraries.  In case you want to download it explicitly (to implement your own client library, for example), you can find the source
+in here.
```

Comment:
```suggestion
Typically, you will not need to download `azure-identity`; it will be downloaded for you as a dependency of the client libraries. In case you want to download it explicitly (to implement your own client library, for example), you can find the source in here.
```